src/app/components/planner/planner.component.scss

```diff
@@ -452,7 +460,23 @@
 		}
 	}
 }
+/*Resizes buttons on mobile for collapsed sidebar, resizes buttons on the larger layout when the browser becomes smaller*/
 
+@media screen and (max-width:560px), screen and (min-width:992px) and (max-width:1300px){
+	.planner-interface {
```

Comment:
While you can do this (and it was done above), the following is equivalent to

```scss
.planner-interface .planner-container .calendar-container .sidebar .sidebar-header .btn {
    font-size: 0.65rem;
}
```

which might be more concise just for future reference. We should probably fix this and the above code.

Reply:
If you think it's best not to resize the buttons, then this code can just be scrapped altogether anyways.

Reply:
We can also get the opinions of @nickbclifford and @AsianPsychoBoy. If we want to scrap the code, then we can close this PR. If you want to redo it, this is still a good issue to solve.

Reply:
Oh, I just meant the SCSS code resizing the buttons (not the rest of it). It's not at all difficult to get one button per line but then it will start creating overlap problems I haven't looked at fixing yet (should be fairly easy).